tests/fixtures/descriptor_tests/with_create_descriptor_hook/env/project.yml

```diff
@@ -0,0 +1,19 @@
+engines:
+  test_engine:
+    location:
```

Comment:
As I said earlier, these are packages that do not exists on the appstore, so failure to download them from our hook will fail bootstrapping.